### PR TITLE
Add linear axis formating

### PR DESCRIPTION
**Pull request by bobhaslett**
Formatting on linear axis and addition of a divisor

### CODE REVIEW

File: src/xLinear.js [hunk 1]
```diff
@@ -32,10 +36,31 @@ export default function () {
...
+
         const xAxis = getAxis(align)
             .tickSize(tickSize)
             .ticks(numTicks)
-            .scale(scale);
+            .scale(scale)
+            .tickFormat(formatNumber);
+
+        function formatNumber(d) {
+            const test4Decimal = Number.isInteger(d / divisor);
```
Comment (aendrew) [inline src/xLinear.js]: I'd rename this variable, using 4 in place of for is confusing, could be interpreted as "test if number has 4 decimal places". A good convention for boolean values is prepend is to the variable name; i.e., isDecimalNumber.

File: src/yLinear.js [hunk 2]
```diff
@@ -26,9 +30,30 @@ export default function () {
...
+        const numberFormat = d3.format(',');
+
         const yAxis = getAxis(align)
             .ticks(numTicks)
-            .scale(scale);
+            .scale(scale)
+            .tickFormat(formatNumber);
+
+        function formatNumber(d) {
+            const test4Decimal = Number.isInteger(d / divisor);
```
Comment (aendrew) [inline src/yLinear.js]: Same here re: variable naming.

File: src/xLinear.js (hunk 1 above)
Comment (bobhaslett) [reply src/xLinear.js]: Those changes now done and new test/build committed

File: src/yLinear.js (hunk 2 above)
Comment (bobhaslett) [reply src/yLinear.js]: Those changes now done and new test/build committed

**Pull request merged**